CONTRIBUTING.md

```diff
@@ -98,7 +95,10 @@ ensure we have a cleaner commit history.
 
 ### Coding style
 
-The majority of the code is written in Python and we generally follow the
+The majority of the code is written in golang, we follow standard golang guidelines
+and run gofmt and goimport tools over the code to maintain formatting.
+
```

Comment:
Might be worth adding Go style guide (closest to it) here https://github.com/golang/go/wiki/CodeReviewComments